platform/windows/display_server_windows.h

```diff
@@ -457,6 +457,7 @@ class DisplayServerWindows : public DisplayServer {
 	virtual int get_screen_count() const;
 	virtual Point2i screen_get_position(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
 	virtual Size2i screen_get_size(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
+    virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
```

Comment:
You have a few too many spaces at the beginning of this line, compared to the other functions:

```suggestion
  virtual float screen_get_refresh_rate(int p_screen = SCREEN_OF_MAIN_WINDOW) const;
```

(Also, please ignore my other (now-deleted) comment. I didn't actually look at the implementation before sending that!)

Reply:
I've changed this to draft for the moment while I fix formatting + linux compiling issues. I will push the fixes and undraft this PR in a little bit.

Reply:
Fixed